internal/grpc/processors/disable_processor.go

```diff
@@ -1,31 +1,30 @@
 package processors
 
 import (
+	"context"
 	"fmt"
 
 	pb "github.com/SAP/remote-work-processor/build/proto/generated"
 )
 
-// Currently remote work processor ID should be in the following format - <cluster>:<app>:<instance>
 type DisableProcessor struct {
+	disableFunc func()
 }
 
-func NewDisableProcessor() DisableProcessor {
-	return DisableProcessor{}
+func NewDisableProcessor(disableFunc func()) DisableProcessor {
+	return DisableProcessor{
+		disableFunc: disableFunc,
+	}
 }
 
-func (p DisableProcessor) Process() <-chan *ProcessorResult {
-	c := make(chan *ProcessorResult)
-	go p.buildClientMessage(c)
+func (p DisableProcessor) Process(_ context.Context) (*pb.ClientMessage, error) {
+	fmt.Println("Disabling work processor...")
```

Comment:
It seems we use both `log.Println` and `fmt.Println`. We should consider doing a refactor in the future to standardize things to a whichever choose to be the more desirable one.

Reply:
I had missed changing these to `log.Print*`